src/main/webapp/app/tutor-exercise-dashboard/tutor-exercise-dashboard.component.html

```diff
@@ -34,35 +34,35 @@
                     <div class="col-8">{{ 'artemisApp.tutorExerciseDashboard.numberOfSubmissionsAssessedByYou' | translate }}</div>
                     <div class="col-4">{{ numberOfTutorAssessments }} ({{ tutorAssessmentPercentage }}%)</div>
                 </div>
-                <div *ngIf="exercise.type !== ExerciseType_PROGRAMMING" class="row mb-1">
+                <div *ngIf="exercise.type !== ExerciseType.PROGRAMMING" class="row mb-1">
                     <div class="col-8">{{ 'artemisApp.tutorCourseDashboard.totalComplaints' | translate }}</div>
                     <div class="col-4">
                         <a routerLink="/complaints" [queryParams]="{ exerciseId: exerciseId, tutorId: tutor?.id }">{{ numberOfComplaints }}</a>
                     </div>
                 </div>
-                <div *ngIf="exercise.type !== ExerciseType_PROGRAMMING" class="row mb-1">
+                <div *ngIf="exercise.type !== ExerciseType.PROGRAMMING" class="row mb-1">
                     <div class="col-8">{{ 'artemisApp.tutorCourseDashboard.complaintsAboutYourAssessments' | translate }}</div>
                     <div class="col-4">{{ numberOfTutorComplaints }}</div>
                 </div>
-                <div *ngIf="exercise.type !== ExerciseType_PROGRAMMING" class="row mb-1">
+                <div *ngIf="exercise.type !== ExerciseType.PROGRAMMING" class="row mb-1">
                     <div class="col-8">{{ 'artemisApp.tutorCourseDashboard.totalMoreFeedbackRequests' | translate }}</div>
                     <div class="col-4">
                         <a routerLink="/more-feedback-requests" [queryParams]="{ exerciseId: exerciseId, tutorId: tutor?.id }">{{ numberOfMoreFeedbackRequests }}</a>
                     </div>
                 </div>
-                <div *ngIf="exercise.type !== ExerciseType_PROGRAMMING" class="row mb-1">
+                <div *ngIf="exercise.type !== ExerciseType.PROGRAMMING" class="row mb-1">
                     <div class="col-8">{{ 'artemisApp.tutorCourseDashboard.moreFeedbackRequestsForYourAssessments' | translate }}</div>
                     <div class="col-4">{{ numberOfTutorMoreFeedbackRequests }}</div>
                 </div>
             </jhi-side-panel>
         </div>
     </div>
 
-    <div class="markdown-preview" *ngIf="tutorParticipationStatus === NOT_PARTICIPATED || exercise.type === ExerciseType_TEXT">
+    <div class="markdown-preview" *ngIf="tutorParticipationStatus === NOT_PARTICIPATED">
```

Comment:
Why was the other condition removed?

Reply:
Before, the assessment instructions used as a collapsable next to the table did not support text exercises, so the instructions were always displayed on top.
Now with the updated assessment instructions, this is no longer necessary.

Reply:
Ok, makes sense.